_po4a-tools/po4a-create-all-targets.sh

```diff
@@ -116,6 +119,8 @@ process_with_po4a () {
             --po "$PO_DIR/$lang/${filename}.po" \
             --localized "$targ_doc" \
             --localized-charset "UTF-8" \
+            --no-deprecation \
```

Comment:
This is to prevent a stream of warning messages - we use po4a in a way that is considered deprecated, but which actually works better for our workflow - we want to separation between .po file updates and their processing into translated files. The 'updated' way of using po4a combines both steps, which we don't want.

Reply:
Hmm. But will we need to change the behaviour any time soon?